libs/components/code-examples/src/lib/modules/action-bars/summary-action-bar/error/example.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, inject } from '@angular/core';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import {
+  SkySummaryActionBarError,
+  SkySummaryActionBarModule,
+} from '@skyux/action-bars';
+import { SkyKeyInfoModule } from '@skyux/indicators';
+
+interface donationSummary {
+  value: number;
+  label: string;
+}
+
+/**
+ * @title Summary action bar with errors
+ * @docsDemoHidden
+ */
+@Component({
+  selector: 'app-action-bars-summary-action-bar-error-example',
+  templateUrl: './example.component.html',
+  imports: [SkyKeyInfoModule, SkySummaryActionBarModule, ReactiveFormsModule],
+})
+export class ActionBarsSummaryActionBarErrorExampleComponent {
+  public formErrors: SkySummaryActionBarError[] = [];
+  public exampleForm: FormGroup;
+  public donationSummary: FormControl<donationSummary[] | null>;
+
+  constructor() {
+    this.donationSummary = new FormControl<donationSummary[] | null>([
+      {
+        value: 250,
+        label: 'Given this month',
+      },
+      {
+        value: 1000,
+        label: 'Given this year',
+      },
+      {
+        value: 1300,
+        label: 'Given all time',
+      },
+    ]);
+    this.exampleForm = inject(FormBuilder).group({
```

Comment:
Both `donationSummary` and `exampleForm` can be defined outside of the constructor.